cmd/config/subcommand/register/files_config.go

```diff
@@ -11,7 +11,7 @@ var (
 
 // FilesConfig containing flags used for registration
 type FilesConfig struct {
-	Version              string `json:"version" pflag:",version of the entity to be registered with flyte which are un-versioned after serialization."`
+	Version              string `json:"version" pflag:",version of the entity to be registered with flyte which are un-versioned after serialization. Random UUID if empty"`
```

Comment:
version of the entity to be registered with flyte which are un-versioned after serialization, By default it generate a random uid for version.

Reply:
updated, thanks